components/spin/index.tsx

```diff
@@ -56,6 +56,14 @@ export default class Spin extends React.Component<SpinProps, SpinState> {
     return !!(this.props && this.props.children);
   }
 
+  componentWillMount() {
```

Comment:
Can we use `componentDidMount`? Because `componentWillMount` will be marked as unsafe in the feature versions of react.

Reply:
Forgot to reply there. The method has been changed to "componentDidMount".